examples/ex9/ex9.cpp

```diff
@@ -74,7 +74,7 @@ int main(int argc, char **argv) {
 
     double dt = 0.0;
     double tMax = 1.0; // max simulation time in seconds
-    double dMax = 0.007; // max displacment in meters
+    double dMax = 0.001; // max displacment in meters
```

Comment:
@enstrophy I want to @RitikaMenghani review of this. I believe that in the abaqus directory, the abaqus benchmark was for 0.007 to keep the strains larger. We want to keep the abaqus benchmark the same loading as this example.

Reply:
Updated in the latest commit.